src/types/flags.ts

```diff
@@ -1,6 +1,6 @@
 export type FlagSet =
   | "Planning permission"
-  | "Listed building consent"
+  | "Works to listed buildings"
```

Comment:
This file is the important change! Once merged to planx-new, we should see all of our flag color bands re-appear as well as _new_ Filters added having updated data values & text set here